app/src/lib/enable-source-maps.ts

```diff
@@ -0,0 +1,60 @@
+import * as Path from 'path'
+import * as Fs from 'fs'
+
+const fileUriToPath = require('file-uri-to-path') as (uri: string) => string
+const sourceMapSupport = require('source-map-support')
+
+// Can't use in dev builds because if the fs module is available source-maps-support always uses that
+// regardless of the environment. We also want the node offset calculations that they do for environment node
+
+export function enableSourceMaps() {
+  sourceMapSupport.install({
+    environment: 'node',
+    handleUncaughtExceptions: false,
+    retrieveSourceMap: function(source: string) {
+      if (!source.endsWith('renderer.js') && !source.endsWith('main.js') && !source.endsWith('shared.js')) {
```

Comment:
What do you think about pulling these out into a const array at the top of the file?